src/rules/no-firestore-object-arrays.ts

```
@@ -176,8 +141,206 @@ Prefer:
+            visitNode(body as unknown as TSESTree.Node);
+          }
+          break;
+        }
+        default: {
+          // Only traverse a shallow subset we care about
+          break;
+        }
+      }
+    };
+
+    for (const stmt of (sourceCode.ast.body as unknown as TSESTree.Node[])) {
```

ESLint

[error] Replace `(sourceCode.ast.body·as·unknown·as·TSESTree.Node[])` with `sourceCode.ast.body·as·unknown·as·TSESTree.Node[]`

(prettier/prettier)